cryptoprotocol/src/main/java/org/cryptimeleon/incentivesystem/cryptoprotocol/IncentiveSystem.java

```diff
@@ -351,7 +357,7 @@ public SpendRequest generateSpendRequest(Token token,
 
         /* Enable double-spending-protection by forcing usk and esk becoming public in that case
            If token is used twice in two different transactions, the provider observes (c0,c1), (c0',c1') with gamma!=gamma'
-           Hence, the provider can easily retrieve usk and esk. */
+           Hence, the provider can easily retrieve usk and esk (using the Schnorr-trick). */
```

Comment:
Does this help anyone? Maybe write the "trick" here.

Reply:
Done